prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -2,27 +2,55 @@
+
+* `adUnitId`: This is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
+
+* `accountId`: The `accountId` you use to initialize the Prebid Mobile SDK is the unique identifier your Prebid Server provider assigned to you. This links requests from your app to server-side settings such as timeout, price granularity, and others.
+
+* `adServer`: This parameter specifies the ad server you are using. It is an enum value of the type `Prebid.AdServer`. The possible values include `Prebid.AdServer.DFP` for Google Ad Manager and `Prebid.AdServer.MOPub` for MoPub.
+
+* `host`: If you are using a managed Prebid server service, the SDK is coded with enums containing the URLs for those services, for example `Prebid.Host.RUBICON` and `Prebid.Host.APPNEXUS`. Other Prebid Server managed services exist that don't have enums linking to the URL. You can work with them by entering the appropriate URL. If you are using a self-hosted server, you should enter the URL to your server.
+
+Here is a working example:
+
+```java
+Prebid.init(getApplicationContext(), adUnits, "INSERT-ACCOUNT-ID-HERE", Prebid.AdServer.DFP, Prebid.Host.RUBICON);
+```
+
+You can use Prebid SDK to monetize your app with a custom ad server or even without one. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without the primary ad server and its SDK.
 
 * TOC
 {:toc}
 
 ## Transport API
 
-The default ad server for Prebid's Mobile SDK is GAM. The SDK can be expanded to include support for 3rd party ad servers through the fetchDemand function. This function returns the Prebid Server bidder key/values (targeting keys), which can then be passed to the ad server of choice.
+The default ad server for Prebid's Mobile SDK is GAM. The SDK can be expanded to include support for 3rd party ad servers through the [`fetchDemand`](https://docs.prebid.org/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html#html-banner) function. This function returns the Prebid Server bidder key/values (targeting keys), which can then be passed to the ad server of choice.
 
 In this mode, the publisher will be responsible for the following actions:
 
-* Call fetchDemand with extended targetingDict callback
-* Retrieve targeting keys from the extended fetchDemand function
+* Call `fetchDemand` with extended `targetingDict` callback
+* Retrieve targeting keys from the extended `fetchDemand` function
 * Convert targeting keys into the format for your ad server
 * Pass converted keys to your ad server
 * Render ad with Prebid Universal Creative or custom renderer
```

Comment:
github won't let me comment on lines that weren't touched, but I don't like the statement on line 67
```
The basic integration steps for these ad units you can find on the page for integration using [Original API](/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html).
```
1. Backwards, passive sentence.
2. Missing a "the", but more importantly, it's not clear to me that the reference to the "Original API" is helpful at all. In fact it seems like a distraction. The page this points to is huge. 

Please consider whether line 67 brings any value and kill it if not.

Reply:
thx